content/docs/2.5/scalers/gcp-pub-sub.md

```diff
@@ -15,16 +15,17 @@ This specification describes the `gcp-pubsub` trigger for Google Cloud Platform
 triggers:
 - type: gcp-pubsub
   metadata:
-    subscriptionSize: "5" # Optional - Default is 5
+    mode: "SubscriptionSize" # Optional - Default is SubscriptionSize - SubscriptionSize or OldestUnackedMessageAge
+    value: "5" # Optional - Default is 5 for SubscriptionSize | Default is 10 for OldestUnackedMessageAge
```

Comment:
Need to document the backwards compatibility so cannot remove this

Reply:
Good catch.